components/layout/Footer.tsx

```diff
@@ -143,6 +146,40 @@ function Footer() {
           </Grid>
         </FooterSection>
 
+        <FooterSection title="Find us on socials" sx={{ display: { xs: 0, md: 'block' } }}>
+          <Stack
+            className="social-links"
+            direction="row"
+            spacing={1.5}
+            sx={{ justifyContent: { xs: 'center', md: 'flex-start' } }}
+          >
+            <Link
+              color="inherit"
+              href="https://www.linkedin.com/company/clearviction"
+              className="linkedin-link"
+              aria-label="Click to visit Clearviction's LinkedIn"
+            >
+              <LinkedIn sx={{ fontSize: 32 }} />
```

Comment:
Any way to use REM here instead of PX for font size?

Reply:
Updated. Set fontSize for the social icons with rem.